project/AppDependencies.scala

```diff
@@ -3,18 +3,16 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"       %% "bootstrap-backend-play-30" % "8.1.0",
+    "uk.gov.hmrc"       %% "bootstrap-backend-play-30" % "8.2.0",
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"        % "1.6.0",
     "com.beachape"      %% "enumeratum-play-json"      % "1.8.0",
     "commons-io"        %  "commons-io"                % "2.11.0"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"            %% "bootstrap-test-play-30"  % "8.1.0"   % Test,
+    "uk.gov.hmrc"            %% "bootstrap-test-play-30"  % "8.2.0"   % Test,
     "org.mockito"            %% "mockito-scala-scalatest" % "1.17.30" % Test,
-    "com.typesafe.play"      %% "play-test"               % "2.8.18"  % Test,
```

Comment:
This was a Bobby violation, so I removed it entirely. Something somewhere is still depending on the one in `org.playframework`, version 3. So I really shouldn't have left this in.

![image](https://github.com/hmrc/debt-transformation-stub/assets/141128938/208617a0-5d1c-4d18-a91f-4f2829de6181)